Tasks/Maven/maventask.ts

```diff
@@ -6,15 +6,15 @@ import fs = require('fs');
 import tl = require('vsts-task-lib/task');
 import trm = require('vsts-task-lib/toolrunner');
 
-import pmd = require('./pmdForMaven');
-import ar = require('./analysisResult');
+import pmd = require('./pmdformaven');
+import ar = require('./analysisresult');
 
 // Set up for localization
 tl.setResourcePath(path.join( __dirname, 'task.json'));
 
 // Cache build variables - if they are null, we are in a test env and can use test inputs
 var sourcesDir:string = tl.getVariable('build.sourcesDirectory') || tl.getInput('test.sourcesDirectory');
-var stagingDir:string = tl.getVariable('build.stagingDirectory') || tl.getInput('test.stagingDirectory');
+var stagingDir:string = tl.getVariable('build.artifactStagingDirectory') || tl.getInput('test.artifactStagingDirectory');
```

Comment:
The OR statement logic for picking up a test input (working around an inability to set build variables in a test context - can discuss this offline) is quite specialised, and should really be kept global.